autograder/tests/data/config_files/config.json

```diff
@@ -0,0 +1,36 @@
+
+{
+    "database_details": {
+        "dbname": "submitty_fall22_cptr141"
+    },
+    "course_details": {
+        "course_name": "Fundamentals of Programming I",
+        "course_home_url": "https://class.wallawalla.edu/d2l/home/343700",
```

Comment:
Make this a generic class ... something like "CS1 Git Problem Example". Remove all WWU references. This goes for the whole PR.

Reply:
Changes have been applied.